src/flux/student-dashboard.coffee

```diff
@@ -0,0 +1,56 @@
+{CrudConfig, makeSimpleStore, extendConfig} = require './helpers'
+{TimeStore} = require './time'
+_ = require 'underscore'
+moment = require 'moment'
+
+addEvents = (hash, events) ->
+  for event in events
+    week = hash[ moment(event.due_at).startOf('isoweek').format("YYYYww") ] ||= []
```

Comment:
That format is actually not used for display.  It's a combination of the year and week number that's used as a key to group events together by week.